src/molecules/Select/Options.tsx

```diff
@@ -44,6 +44,12 @@ const Options = <T extends {}>({
         width="auto"
       >
         <Menu bordered data-qaid={`${qaId}-menu`}>
+          {options.length === 0 && (
+            <MenuItem disabled data-qaid={`${qaId}-empty`}>
+              No options...
```

Comment:
Could be editable or at least translated?

Reply:
could be. but isn't